builder/image.d/make_repart_partition

```diff
@@ -223,22 +230,41 @@ elif [ "$cryptsetup" = 1 ]; then
 	EOF_GENERATOR
 	chmod +x "$target/etc/systemd/system-generators/repart-$repart"
 else
+	dev_path="/dev/disk/by-partuuid/$repart_uuid"
+
 	cat > "$target/etc/repart.d/1.$repart.conf" <<-EOF
 	[Partition]
 	UUID=$repart_uuid
 	Type=$repart_type
 	Weight=$weight
 	Format=$fs
 	CopyFiles=/etc/repart.templates/$repart_uuid:/
+	Encrypt=$tpm2
```

Comment:
this value in the repart.conf should be conditional on `$tmp2` being set, otherwise non tpm, plain text, repart partitions won't work anymore.

Reply:
`$tpm2` defaults to "off", which should be sufficient for non-encrypted partitions

Reply:
:+1: